src/components/VolumeSlider.js

```diff
@@ -1,5 +1,12 @@
 import React from 'react'
 import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+VolumeSlider.propTypes = {
+  label: PropTypes.string.isRequired,
+  masterVolume: PropTypes.number.isRequired,
```

Comment:
Here as well: Failed prop type: Invalid prop `masterVolume` of type `string` supplied to `VolumeSlider`, expected `number`.